### ISSUE TITLE
update from 1.58.3 to 1.59.2

### ISSUE THREAD

**Issue by licks0re** [hzi-braunschweig/SORMAS-Docker#240]
Dear team,

I'm updating through git, everything seems ok, containers launch etc, and when I'm logging I have a red message : "Fatal error - please contact your administrator". And nothing else.

I tried to upgrade a lot of different ways, but no luck.

I installed the unaccent extention before the update, by upping the postgres container, ssh in it, su postgres, psql, enable the extention check with \dx, it's there and running properly.

But in any way, it won't work.

After that, if I drop the sormas & sormas_audit DB, recreate them and upload a backup in them, it won't work, generate a lot of error and fail.

Anybody experienced and failure during this particular update?
Cheers,
Gui

**Comment by asohh** [comment]
hey,

could you please add the stack trace of sormas when the error occurs.

**Comment by licks0re** [comment]
Hello Adrian, 

Thanks for answering quickly.
Find attached the full trace of the error. It occurs when I try to login.

I see some unaccent extention error, but I confirm the extention is there : 

```
[root@sormasdev SORMAS-Docker]#  psql -h localhost -p 5432 -U postgres
Password for user postgres:
psql (10.17, server 10.16)
Type "help" for help.

postgres=# \dx
                         List of installed extensions
   Name   | Version |   Schema   |                 Description
----------+---------+------------+---------------------------------------------
 plpgsql  | 1.0     | pg_catalog | PL/pgSQL procedural language
 unaccent | 1.1     | public     | text search dictionary that removes accents
(2 rows)

postgres=#

```

So ... I don't know! Any idea will be welcome ! 

Cheers,
Gui

![image](https://user-images.githubusercontent.com/9083552/123977901-b135b500-d9bf-11eb-8534-dc265ec3efde.png)



[error_log.txt](https://github.com/hzi-braunschweig/SORMAS-Docker/files/6741637/error_log.txt)

**Comment by licks0re** [comment]
Good morning,

Ok, the problem is coming from the unaccent.

I added it as postgres user by just doing a create extention unaccent; which is not working, the extention won't show up for sormas_user.

I took a db dump, added a line : "CREATE EXTENSION IF NOT EXISTS unaccent WITH SCHEMA public;" , imported it to sormas DB and ... and it's working properly!

Thx Adrian for the quick answer anyway!

I close the issue! ;)
